cloudwatch-snowflake/tracker/utils/cloudwatch.js

```diff
@@ -0,0 +1,159 @@
+    ReturnData: true
+  }))
+}
+
+const responseTimesQueries = ({ projectName, loadBalancerName, environment }) => {
+  return PERCENTILES.map(percentile => ({
+    Id: `${percentile}_${formatQueryId(loadBalancerName)}_${projectName}_${environment}`, /* /^[a-z][a-zA-Z0-9_]*$./ */
+    MetricStat: {
+      Metric: {
+        Dimensions: [
+          {
+            Name: 'LoadBalancer',
+            Value: loadBalancerName
+          },
+        ],
+        MetricName: 'TargetResponseTime', /* required */
+        Namespace: 'AWS/ApplicationELB', /* required */
+      },
+      Period: constants.PERIOD,
+      Stat: percentile, /* required */
+      Unit: 'Seconds' // If we set it as Milliseconds, the value returned will be undefined
+    },
+    ReturnData: true
+  }))
+}
+
+const errorCountsQueries = ({ projectName, loadBalancerName, environment }) => {
+  return {
+    Id: `errorCount_${formatQueryId(loadBalancerName)}_${projectName}_${environment}`, // /^[a-z][a-zA-Z0-9_]*$./
+    MetricStat: {
+      Metric: {
+        Dimensions: [
+          {
+            Name: 'LoadBalancer',
+            Value: loadBalancerName
+          },
+        ],
+        MetricName: 'HTTPCode_Target_5XX_Count', /* required */
+        Namespace: 'AWS/ApplicationELB', /* required */
+      },
+      Period: constants.PERIOD,
+      Stat: 'Sum', /* required */
+      Unit: 'Count'
+    },
+    ReturnData: true
+  }
+}
+
+const performanceMetricQueries = ({ serviceName, clusterName, projectName, loadBalancerName, environment }) => {
+  return [
+    ...ecsMetricQueries({ serviceName, clusterName, projectName, environment }),
+    ...responseTimesQueries({ projectName, loadBalancerName, environment }),
+    errorCountsQueries({ projectName, loadBalancerName, environment })
+  ]
+}
+
+const recordRows = ({ dataPoints, params }) => {
+  return dataPoints.map(data => {
+    const queryParam = params.MetricDataQueries.find((param) =>
+      param.Id === data.Id
+    )
+
+    const [projectName, environment] = queryParam.Id.split("_").slice(-2)
+    const unit = queryParam?.MetricStat.Unit
+
+    const metricStat = queryParam?.MetricStat
+
+    return {
+      project_name: projectName,
+      environment: environment,
+      region: constants.AWS_REGION,
+      metric_name: metricStat?.Metric?.MetricName,
+      dimensions: metricStat?.Metric?.Dimensions,
+      start_time: params.StartTime.toISOString(),
+      end_time: params.EndTime.toISOString(),
+      stat: metricStat?.Stat,
+      value: data.Values[0] || (unit === 'Count' ? 0 : "Null"),
+      unit,
+      created_at: new Date().toISOString()
+    }
+  })
+}
+const getCloudwatchData = async () => {
+  const resourcesData = JSON.parse(constants.RESOURCES_DATA)
+  const metricDataQueries = resourcesData.flatMap((data) => performanceMetricQueries(data))
+
+  const prevHour = dateUtil.previousHour()
+  const params = {
+    MetricDataQueries: metricDataQueries,
+    StartTime: dateUtil.oneHourBefore(prevHour),
```

Comment:
I admire the simplicity of this solution, but I thought we were generally interested in importing historical data, first, to have a lot of data already. Could you double-check with Fried, in case that was his expectation?

Reply:
@swiknaba. 
Yes, we can write a script to export the data based on our needs.
Different project might have different needs (i.e. different time range) so the script can be vary.